### PR TITLE
Adds UI for Links with trace ids

### PR DESCRIPTION
**Pull request by zeagord**
Adds the UI, service endpoint and the integration

TODO: UI integration tests which will done when we actually implement this feature.

### CODE REVIEW

**Comment by drolando** [comment]
Is there a related issue I can read to understand what you're trying to do?

**Comment by zeagord** [comment]
This is related to #1206

**Comment by codefromthecrypt** [comment]
wanna pop in some screen shots?

**Comment by zeagord** [comment]
Here's the screen capture of the feature.
qch7pkts9s

File: zipkin2/src/main/java/zipkin2/storage/QueryRequest.java [hunk 1]
```diff
@@ -98,6 +98,22 @@ public int limit() {
     return limit;
   }
 
+  /** Parent service name to fetch for the link dependency */
+  @Nullable
+  public String parentServiceName(){
```
Comment (llinder) [inline zipkin2/src/main/java/zipkin2/storage/QueryRequest.java:103]: From what I recall we decided to use a separate query request for this called `DependencyQueryRequest`.

**Pull request merged**